blendergltf.py

```diff
@@ -458,20 +489,15 @@ def export_mesh(me):
                 if i > max_vert_index:
                     max_vert_index = i
 
-            # Triangulate each polygon if necessary
-            if poly.loop_total == 3:
-                prim += indices
-            elif poly.loop_total > 3:
-                for i in range(poly.loop_total-2):
-                    prim += (indices[-1], indices[i], indices[i + 1])
-            else:
-                raise RuntimeError(("Invalid polygon with {} "
-                                    "vertices.").format(poly.loop_total))
+            prim += triangulate(indices)
```

Comment:
Hmm, I never considered the performance aspect of this. Anyways, I'll inline it with another commit.